tests/item/item_api.cpp

```diff
@@ -148,43 +217,70 @@ void test_item() {
   constexpr size_t nMethodsCount = getter::method_cnt;
 
   /* allocate and clear host buffers */
-  std::array<std::array<bool, nSize>, nMethodsCount> dataOut;
-  std::for_each(dataOut.begin(), dataOut.end(),
-                [](std::array<bool, nSize>& arr) { arr.fill(false); });
+  std::array<std::array<bool, nSize>, nMethodsCount> apiData;
+  std::for_each(apiData.begin(), apiData.end(),
+                [](std::array<bool, nSize>& arr) { arr.fill(true); });
+
+  std::array<std::array<bool, nSize>, nMethodsCount> typeData;
+  std::for_each(typeData.begin(), typeData.end(),
+                [](std::array<bool, nSize>& arr) { arr.fill(true); });
 
-  std::vector<int> dataOutDeprecated(nSize);
+  std::vector<int> apiDataDeprecated(nSize);
```

Comment:
Just curious: why a `vector` here and `array` elsewhere?

Reply:
I prefer to use vectors rather than arrays, but they are not suitable with bool type. If it bothers you I can replace all with arrays